### DISCUSSION TITLE
Docker container monitoring only show half command?

### DISCUSSION THREAD

**Discussion by martadinata666** [nicolargo/glances#1898] [Q&A]
Im using glances to monitor my docker installation. But i noticed that the container only show half command. Either it the ENTRYPOINT or CMD. If the container doesn't have ENTRYPOINT it will use CMD and vice versa. 
Full command should be ENTRYPOINT + CMD 

![command](https://res.cloudinary.com/dedyms/image/upload/v1626408056/issues/2021-07-16-105813_1342x675_scrot_c7ojou.png)

Is this desired?
Like my personal custom image using ENTRYPOINT ```["/usr/bin/tini", "--"]```, and the CMD itself ```["mpd", "--stdout", "--no-daemon", "./mpd.conf"]``` the right glances command column should be ```[ "/usr/bin/tini -- mpd --stdout --no-daemon ./mpd.conf"]```

**Comment by nicolargo** [comment]
Enhancement request created ==> https://github.com/nicolargo/glances/issues/1912
